### ISSUE TITLE
wrong link in "Next" button of Smart Contracts page 

### ISSUE THREAD

**Issue by Imod7** [substrate-developer-hub/substrate-docs#1975]
### Broken Link

The "Next" button in this page 
https://docs.substrate.io/tutorials/smart-contracts/

leads to 
https://docs.substrate.io/quick-start/

### Expected Location

I think it should lead here : 
https://docs.substrate.io/tutorials/smart-contracts/prepare-your-first-contract/

### Context and Origin

Also, "Smart Contracts" menu link does not appear in the Sidecar

<img width="290" alt="Screenshot 2023-04-27 at 12 53 48" src="https://user-images.githubusercontent.com/12569221/234841896-87f4f9a3-5093-45a6-a95c-1f2bfea49e08.png">

**Comment by jonnysmillie** [comment]
Yes, for some reason the page has been kept but removed from the nav.yaml file.

A reference to it should probably be added back into the nav.yaml in the relevant place then this issue will be resolved.

**Comment by sacha-l** [comment]
Ran into this issue too - was going to flag but found this issue already open. 

<img width="1218" alt="Screenshot 2023-05-03 at 3 47 29 PM" src="https://user-images.githubusercontent.com/23283108/235935419-5cc11901-51a7-470c-bc0d-7c31e1ff235e.png">
